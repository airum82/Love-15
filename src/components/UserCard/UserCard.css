.user-card {
  background-color: rgba(229, 149, 0, 1);
  width: 60%;
  margin: 0 auto;
}
