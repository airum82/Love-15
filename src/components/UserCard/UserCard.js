import React from 'react';

export const UserCard = ({ username }) => {
  return (
    <div className="user-card">
      <h4>{username}</h4>
    </div>
  )
}