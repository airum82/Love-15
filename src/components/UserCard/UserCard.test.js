import React from 'react';
import { UserCard } from './UserCard';
import { shallow } from 'enzyme';

describe('UserCard', () => {

  it('should match snapshot', () => {
    const wrapper = shallow(
     <UserCard
      username={'melissa'} 
    />);
    expect(wrapper).toMatchSnapshot();
  })
})