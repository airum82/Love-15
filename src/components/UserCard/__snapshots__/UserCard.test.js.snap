// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserCard should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    username="melissa"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h4>
        melissa
</h4>,
      "className": "user-card",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "melissa",
      },
      "ref": null,
      "rendered": "melissa",
      "type": "h4",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h4>
          melissa
</h4>,
        "className": "user-card",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "melissa",
        },
        "ref": null,
        "rendered": "melissa",
        "type": "h4",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
